samples/docker-domain/weblogic-deploy.zip.download

```diff
@@ -0,0 +1,5 @@
+# Download the WebLogic Deploy Tool installer to the sample directory.
+#
+#  - wget https://github.com/oracle/weblogic-deploy-tooling/releases/download/weblogic-deploy-tooling-0.11/weblogic-deploy.zip
+#
+  weblogic-deploy.zip
```

Comment:
Is there a checksum for this zip?

Reply:
Github doesn't appear to offer checksum capabilities.  We could put something in the release notes, but seems kinda odd.